meson.build

```diff
@@ -196,6 +196,12 @@ else
 endif
 
 # Handle PCRE2
+cpu_jit_supported = [ 'aarch64', 'arm', 'mips', 'mips64', 'ppc', 'ppc64', 'riscv32', 'riscv64', 's390x', 'x86', 'x86_64' ]
+pcre2_jit_supported = host_machine.cpu_family() in cpu_jit_supported and cc.get_id() != 'tcc'
```

Comment:
shouldn't this be `target_machine` ?

Reply:
No:
> build machine is the computer that is doing the actual compiling.
    host machine is the machine on which the compiled binary will run.
    target machine is the machine on which the compiled binary's output will run, only meaningful if the program produces machine-specific output.
>
> The tl/dr summary is the following: if you are doing regular cross compilation, you only care about [build_machine](https://mesonbuild.com/Reference-manual_builtin_build_machine.html) and [host_machine](https://mesonbuild.com/Reference-manual_builtin_host_machine.html)

see https://mesonbuild.com/Cross-compilation.html
`host_machine` is correct, `target_machine` is for compiling compilers.